platform-hub-api/app/models/contact_list.rb

```diff
@@ -0,0 +1,41 @@
+class ContactList
+
+  HASH_RECORD_KEY_POSTFIX = '_contact_list'.freeze
+  HASH_RECORD_KEY_POSTFIX_REGEX = /#{HASH_RECORD_KEY_POSTFIX}$/
+
+  class << self
+    def find id
+      key = "#{id}#{ContactList::HASH_RECORD_KEY_POSTFIX}"
+      hash_record = HashRecord.contact_lists.find_or_create_by!(id: key) do |r|
+        r.data = { email_addresses: [] }
+      end
+      ContactList.new hash_record
+    end
+  end
+
+  # Needed for active_model_serializers to work :(
+  alias :read_attribute_for_serialization :send
```

Comment:
Unfortunately so – when the serializer runs, it looks for a `read_attribute_for_serialization` method on the object (which for AR objects is automatically mixed in). It's yucky and unfortunate.